ONSdigital/census-rm-qid-batch-runner
# Increase batch quantities

# Motivation and Context
The amount of unaddressed materials that we want to print, has increased.

# What has changed
Increased the quantities.

# How to test?
Try running the batch - it should send bazillions of messages, and the big print files (or don't... this is going to be rehearsed A LOT)

# Links
Trello: https://trello.com/c/PMAQuMcf
